src/main/java/org/folio/list/services/ListService.java

```diff
@@ -153,7 +153,8 @@ public Optional<ListDTO> updateList(UUID id, ListUpdateRequestDTO request) {
         timer.start(TimedStage.TOTAL);
         importListContentsFromAsyncQuery(list, getCurrentUser(), request.getQueryId(), timer);
       }
-
+      ListVersion previousVersions = new ListVersion();
```

Comment:
updating the list in the version table as well (after the version is incremented)